README.md

```diff
@@ -2,16 +2,18 @@
 
 Welcome to Hartwig Medical Foundation's documentation site! 
 
-This page is meant as a resource to researchers who have requested access to our data. To make a data request please visit our [main site](https://www.hartwigmedicalfoundation.nl/en/applying-for-data/)
+This page is meant as a resource to researchers who have requested access to our data. To get started on a data access request please visit our [website](https://www.hartwigmedicalfoundation.nl/en/applying-for-data/).
 
-Here you'll find instructions on data access, data interpretation and data generation methods used within the Hartwig Medical Foundation
+Here you'll find instructions on data access, data interpretation and data generation methods used within the Hartwig Medical Foundation.
 
 Our data is currently hosted on the [Google Cloud Platform](https://cloud.google.com/), so please start by reviewing the account creation process. 
 We recommend researchers start the process of account acquisition while their application is reviewed to avoid any delays once the request is approved.
 
 ## Contents
 
-* [Getting a Google Cloud Platform Account](getting-a-Google-account.md): This document contains instructions on how to set up a Google Account for accessing the HMF data set.
-* [Accessing HMF Data through Google Cloud Platform](accessing-hmf-data-through-gcp.md): This document contains instruction on how to use GCP to download or process HMF data. 
-* [HMF Data Access Request Guide](data-access-request-guide.md): This document contains information regarding the files you receive when your Data Access Request is approved. How are the samples selected, what formats will you receive.
-* [HMF Data Access Request Methods](data-access-request-methods.md): This document contains the methods that were used to generate the data that is serviced through Data Access Requests.
+* [Instructions on completing your Data Access Request application](data-access-request-application): A walkthrough of the application that needs to be filled out when applying for Hartwig data.
+* [Getting a Google Cloud Platform Account](getting-a-Google-account.md): Instructions on how to set up a Google Account for accessing the Hartwig Medical Foundation data set.
+* [Accessing Hartwig Data through Google Cloud Platform](accessing-hartwig-data-through-gcp.md): Instruction on how to use GCP to download or process HMF data. 
+* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved. How are the samples selected, what formats will you receive.
```

Comment:
Sentence fragment. " Data Access Request is approved: how are samples are selected; which formats will you receive." would fix.

Reply:
```suggestion
* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved: how samples are selected; which formats will you receive.
```

Reply:
```suggestion
* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved: how are samples selected; which formats will you receive.
```